plugins/panels/similarity_search/operators.py

```
@@ -236,6 +242,34 @@ def _compute_combined_query(
 
         return combined
 
+    @staticmethod
+    def _embed_query_image(ctx):
```

Ruff (0.15.9)

[warning] Missing return type annotation for staticmethod `_embed_query_image`

(ANN205)